src/main/java/com/salesforce/dynamodbv2/mt/util/SequenceNumber.java

```diff
@@ -0,0 +1,56 @@
+package com.salesforce.dynamodbv2.mt.util;
+
+import com.amazonaws.services.dynamodbv2.model.Record;
+import java.math.BigInteger;
+import java.util.Objects;
+
+final class SequenceNumber implements Comparable<SequenceNumber> {
```

Comment:
Probably worth getting rid of this class and just using BigInteger directly instead. (Originally, this class was storing more stuff, but I got rid of that.)